README.md

```diff
@@ -1 +1,30 @@
-# Hello-World
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello Microverse
+
```

Comment:
- [ ] Kindly add a description for your project here, so anyone who sees the readme knows what this project is.